### PR TITLE
Predict script

### PR DESCRIPTION
**Pull request by xFiveRivers**
Reference #19 

Completed script for developing model/prediction. Script takes a while to run so please let me know any changes you want to make in bulk so the script does not need to be run many times.

### CODE REVIEW

**Comment by erictsai1208** [comment]
Looks good. I think maybe we can also include a DataFrame of the $R^2$ score as well as MAPE of the model evaluated on the test set. Then, we can save that DataFrame to a csv file for future use so we don't need to rerun this entire script since it takes so long.

File: src/prediction_model.py [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+    cross_val_results_df.to_csv(results_dir + "model_comparison_table.csv")
+
+    # Fit the optimal RFR model
+    pipe_rfr_opt.fit(X_train, y_train)
+
+    # Create dataframe for actual scores from the testing data and the predicted scores from the model
+    results_dict = {
+        "Actual Scores": y_test,
+        "Predicted Scores": pipe_rfr_opt.predict(X_test)
+    }
```
Comment (erictsai1208) [inline src/prediction_model.py:251]: Using `results_dict` here, we can just call `mean_absolute_percentage_error` and call `.score` on the test set again and save that to a DataFrame.

File: src/prediction_model.py [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+    results_df = pd.DataFrame(results_dict)
+
```
Comment (erictsai1208) [inline src/prediction_model.py:260]: Using `results_dict` here, we can just call `mean_absolute_percentage_error` and call `.score` on the test set again and save that to a DataFrame.

**Comment by xFiveRivers** [comment]
I actually use multiple scoring methods and save that to a csv, then we can just filter the ones we want to look at specifically.

*Edit* Ah my mistake I see what you mean. Specifically on the test set, yeah thats a good idea.

**Review by marianagyby — APPROVED** [review]
Looks good!

**Review by erictsai1208 — APPROVED** [review]
Looks good to merge.

**Pull request merged**